config/routes.rb

```diff
@@ -27,10 +27,11 @@
 
   get '/pet_applications/new', to: 'pet_applications#new'
   post '/pet_applications', to: 'pet_applications#create'
+  get '/pet_applications/:id', to: 'pet_applications#show'
```

Comment:
it occured to me that we could be using `/applications` to: `pet_applications#action` because the action controller word duplication isn't an issue on the website side of things. but this is not a functional issue and i'd only go back and refactor later if we had time at that point